srcpkgs/WebStorm/template

```diff
@@ -1,39 +1,50 @@
 # Template file for 'WebStorm'
 pkgname=WebStorm
-version=2020.1.4
+version=2024.2.1
 revision=1
-archs="i686 x86_64"
-depends="jetbrains-jdk-bin"
+archs="x86_64 aarch64"
+depends="virtual?java-environment"
```

Comment:
`webstorm` (and other jetbrains ide's)  depends on the patched openjdk runtime called [JetBrainsRuntime](https://github.com/JetBrains/JetBrainsRuntime). it's located in `jbr` directory which you copied to targetdir.

you either need to remove `depends` section, OR update`jetbrains-jdk-bin` package (not sure about this one) and remove `jbr` dir. the original maintainer created `jetbrains-jdk-bin` for a reason.

P.S. IMO, closing this PR and using jetbrains toolbox to update their products is an option too.

Reply:
Intellij or also rubymine doesn't use jetbrains jdk dependency and they work fine.